Python/Product/VSInterpreters/PackageManager/PipPackageManager.cs

```diff
@@ -444,6 +444,15 @@ CancellationToken cancellationToken
                                             .Where(p => p.IsValid)
                                             .OrderBy(p => p.Name)
                                             .ToList();
+
+
+                                        if(packages.Count() == 0 && proc.StandardErrorLines.Any()) {
+                                            Debug.WriteLine("Failed to run pip to collect packages");
+                                            foreach (var line in proc.StandardErrorLines) {
+                                                Debug.WriteLine(line);
+                                            }
+                                            packages = null;
```

Comment:
below there is a case for null packages.
   if (packages == null) {
                        // Pip failed, so return a directory listing